dotnet/build-packages.sh

```diff
@@ -27,6 +27,8 @@ echo "Copying files for development version"
 cp -r ./dotnet ./dev-dotnet
 
 echo "Versioning release files"
+sed -i "s/vFOLDERUNKNOWN/v${RELEASE_VERSION_FILE}/g" dotnet/content/Agent/datadog.yaml
```

Comment:
Looks like these changes also need to be ported to `java/build-packages.sh`

Reply:
I didn't hit `ctrl+s` on that file ;(